test_pts.py

```diff
@@ -0,0 +1,33 @@
+# Generates test points for FFT
```

Comment:
You should specify in this file what version of python, what pip requirements are needed and probably just make this an executable with a shebang.